crates/re_ui/src/lib.rs

```diff
@@ -862,7 +863,7 @@ impl ReUi {
     ) {
         let visuals = ui.style().interact(response);
 
-        let extent = rect.size().min_elem();
+        static TRIANGLE_SIZE: f32 = 8.0;
```

Comment:
Why is this one hardcoded? Why is it not `collapsing_triangle_size().min_size()` or something?

Reply:
This whole implementation of the triangle is kind of weird to begin with, but from the UX perspective, that triangle has a given, fixed width (I would be a 14x14px icon, if we didn't have to rotate it), regardless of where we decide to draw it. So hardcoding its size next to the coordinates is actually more correct than what we did before.

It would actually make even more sense to "bake" that scale into the coordinate table that's just after but I couldn't be bothered updated those. In the future, we may even move the whole thing a some data file (SVG—but we don't want the deps/overhead, or even as glyphs in a custom font).

As for `collapsing_trangle_size()`, this function is about the "layout space" that should be allocated for that triangle—not what size it should have.

Adding some comments to clarify all that ☝🏻